app/services/idv/usps_mail.rb

```diff
@@ -16,6 +17,12 @@ def any_mail_sent?
       user_mail_events.any?
     end
 
+    def most_recent_otp_expired?
+      return false unless any_mail_sent?
+
+      user_mail_events.first.updated_at < USPS_CONFIRMATION_WINDOW_DAYS.days.ago
```

Comment:
Maybe not in this PR, but I think it's worth it to order `user_mail_events` in ASC order because everywhere it's being used, we're naming things the opposite of the meaning. For example, here we are referring to the most recent event, but we are fetching it by calling `first`, which is counterintuitive. I would expect it to be `last`. Same with `updated_within_last_month?`. I would expect it to refer to `first` since it's the oldest mail event that is compared to `MAIL_EVENTS_WINDOW_DAYS`. Thoughts?